### PR TITLE
Feat/update docs

### PR DESCRIPTION
**Pull request by MiroslavRepka**
This PR updates and cleans up the docs & README. 

For the cleanup, I used markdown lint, to show me issues, trailing spaces, etc. These are now resolved.

I also changed the main example manifest as it felt too cluttered, let me know if you like the change.

Finally, I added a simple example manifest for all cloud providers.

### CODE REVIEW

File: docs/claudie-workflow/claudie-workflow.md [hunk 1]
```diff
@@ -54,56 +61,64 @@ Context box is Claudie's "control unit". It holds pending configs, which need to
...
 ## Scheduler
+
 Scheduler brings the infrastructure to a desired the state based on the manifest contained in the config that is received from Context-box.
 
 Scheduler also monitors the health of current infrastructure and manages any operations based on actual health state (e.g. replacement of broken nodes, etc. *[work in progress]*).
 
 ### API
-```
+
+```sh
```
Comment (bernardhalas) [inline docs/claudie-workflow/claudie-workflow.md:89]: Doesn't look like a shell command to me :)

File: docs/claudie-workflow/claudie-workflow.md [hunk 2]
```diff
@@ -54,56 +61,64 @@ Context box is Claudie's "control unit". It holds pending configs, which need to
...
 
-
 ## Builder
+
 Builder aligns the current state of the infrastructure with the desired state. It calls methods on `terraformer`, `ansibler`, `kube-eleven` and `kuber` in order to manage the infrastructure. It follows that Builder also takes care of deleting nodes from a kubernetes cluster by finding differences between `desiredState` and `currentState`.
 
 ### API
-```
+
+```sh
```
Comment (bernardhalas) [inline docs/claudie-workflow/claudie-workflow.md:105]: Not a shell command here either.

File: docs/claudie-workflow/claudie-workflow.md [hunk 3]
```diff
@@ -189,24 +211,27 @@ Kuber manipulates the cluster resources using `kubectl`.
...
 ## Frontend
+
 Frontend is a layer between the user and Claudie.
 New manifests are added as secrets into the kubernetes cluster where `k8s-sidecar` saves them into Frontend's file system
 and notifies the Frontend service via a HTTP request that the new manifests are now available.
 
 ### API
-```
+
+```sh
```
Comment (bernardhalas) [inline docs/claudie-workflow/claudie-workflow.md:229]: +1

File: docs/input-manifest/example.yaml [hunk 4]
```diff
@@ -268,8 +345,6 @@ loadBalancers:
       dns:
         dns_zone: dns-zone
         provider: cloudflare-1
-        # this hostname will be used for DNS records
-        # keep in mind that DNS zone will be included
```
Comment (bernardhalas) [inline docs/input-manifest/example.yaml:272]: Do we want to remove the comments from the manifest example?

**Review by bernardhalas — CHANGES_REQUESTED** [review]

File: docs/claudie-workflow/claudie-workflow.md (hunk 1 above)
Comment (MiroslavRepka) [reply docs/claudie-workflow/claudie-workflow.md:89]: Markdown linter did not like a code block without specified formatting. I just used a `sh` to silence the warnings, can remove them if we want.

File: docs/input-manifest/example.yaml (hunk 4 above)
Comment (MiroslavRepka) [reply docs/input-manifest/example.yaml:272]: I moved the comments from being all over the place to somewhere where it made more sense. In this case, they can be found [here](https://github.com/Berops/claudie/blob/feat/update-docs/docs/input-manifest/example.yaml#L320).

File: docs/claudie-workflow/claudie-workflow.md (hunk 1 above)
Comment (bernardhalas) [reply docs/claudie-workflow/claudie-workflow.md:89]: Ah, I see. However I wouldn't put this sentence into a code-block.

File: docs/input-manifest/example.yaml (hunk 4 above)
Comment (bernardhalas) [reply docs/input-manifest/example.yaml:272]: Thanks, I haven't realized that.

**Review by MarioUhrik — APPROVED** [review]

**Review by bernardhalas — APPROVED** [review]

**Comment by bernardhalas** [comment]
LGTM, thank you!

**Pull request merged**